### ISSUE TITLE
[Bug]: Can't delete user (Please specify a value for 'urls')

### ISSUE THREAD

**Issue by Indigo744** [piwik/piwik#21396]
### What happened?

As a super user, if I go to **Administration** > **System** >**Users** and I try to delete a user, my password is requested and then I get an error message 

> Please specify a value for 'urls'.

![image](https://github.com/matomo-org/matomo/assets/7137528/58b1aa7b-e53b-4668-9776-215adb1aa4b8)

This happens if I use the "bin" icon on the right of the user, or if I use the bulk delete for the user.

### What should happen?

The user should be deleted, of course 😺 

### How can this be reproduced?

See above.

### Matomo Version

Matomo 4

### Matomo Patch or Minor Version

4.15.1

### PHP Version

8.1.23

### Server Operating System

Debian

### What browsers are you seeing the problem on?

Microsoft Edge

### Computer Operating System

Windows 10

### Validations

- [X] Read our [Contributing Guidelines](https://github.com/matomo-org/matomo/blob/5.x-dev/CONTRIBUTING.md).
- [X] Follow our [Security Policy](https://github.com/matomo-org/matomo/blob/5.x-dev/SECURITY.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating duplicates.
- [X] The provided steps to reproduce is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the Bug.

**Comment by sgiehl** [comment]
@Indigo744 Do you have any additional plugins installed?

**Comment by Indigo744** [comment]
I have 49 Core (34 Active), 6 Official (5 active).

Here is the plugin list (taken from Diagnostic tab):
```

#### Plugins Activated:
 API, Actions, Bandwidth 4.0.5, BulkTracking, CoreAdminHome, CoreConsole, CoreHome, CorePluginsAdmin, CoreUpdater, CoreVisualizations, CoreVue, CustomDimensions, CustomVariables 4.1.3, DBStats, Dashboard, DevicePlugins, DevicesDetection, Diagnostics, Events, GeoIp2, ImageGraph, Insights, Installation, Intl, IntranetMeasurable, InvalidateReports 4.1.1, LanguagesManager, Live, Login, Marketplace, Monolog, Morpheus, MultiSites, PagePerformance, PrivacyManager, Proxy, Referrers, SEO, ScheduledReports, SecurityInfo 4.0.2, SegmentEditor, SitesManager, Transitions, TreemapVisualization 4.0.2, TwoFactorAuth, UserCountry, UserCountryMap, UserId, UserLanguage, UsersManager, VisitFrequency, VisitTime, VisitorInterest, VisitsSummary, WebsiteMeasurable, Widgetize

#### Plugins Deactivated:
 Annotations, Contents, CustomJsTracker, Ecommerce, Feedback, Goals, Heartbeat, MobileAppMeasurable, MobileMessaging, Overlay, ProfessionalServices, Provider 4.0.5, Resolution, RssWidget, TagManager, Tour

#### Plugins Invalid:
 
```

**Comment by bx80** [comment]
Hi @Indigo744, unfortunately I've not been able to recreate this issue. 

A few more questions that might help us find the cause:
- Does this happen with any user or just one particular user account? 
- If you create a test user account and then tried to delete it does that work?
- Would it be possible to test the delete functionality with a different browser?

Thanks for taking the time to report this and provide follow up info! :+1:

**Comment by Indigo744** [comment]
Hello,

It was happening for two different users (I didn't try others, as they were in use).

But I went back to try again and... this time it works?
![image](https://github.com/matomo-org/matomo/assets/7137528/fa81664c-9044-4458-af23-bc7fac2ea40c)

I was able to delete the two users that I couldn't earlier.

I really don't understand how or why. Matomo was not updated and this is the same browser...

I'm really sorry to have wasting your time on this. Thank you for your time!
